docs/ref/pgcopydb_copy.rst

```diff
@@ -473,8 +473,7 @@ Now, first dump the schema:
    14:28:50 22 INFO   Dumping database into directory "/tmp/pgcopydb"
    14:28:50 22 INFO   Using pg_dump for Postgres "16.1" at "/usr/bin/pg_dumpall"
    14:28:50 22 INFO   Exported snapshot "00000003-00000022-1" from the source database
-   14:28:50 22 INFO    /usr/bin/pg_dump -Fc --snapshot 00000003-00000022-1 --section pre-data --file /tmp/pgcopydb/schema/pre.dump 'postgres://pagila@source/pagila?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60'
-   14:28:51 22 INFO    /usr/bin/pg_dump -Fc --snapshot 00000003-00000022-1 --section post-data --file /tmp/pgcopydb/schema/post.dump 'postgres://pagila@source/pagila?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60'
+   14:28:50 22 INFO    /usr/bin/pg_dump -Fc --snapshot 00000003-00000022-1 --section pre-data --section post-data --file /tmp/pgcopydb/schema/schema.dump 'postgres://pagila@source/pagila?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60'
 
 Now restore the pre-data schema on the target database, cleaning up the
 already existing objects if any, which allows running this test scenario
```

Comment:
Did you check that this still works? I suppose `make tests/pagila-multi-steps` is as good a test as any. Again, it might be worth replacing these output samples entirely with the new commands.

Reply:
It seems to work. On the other hand, when I follow the commands on this page as follows:
```bash
pgcopydb dump schema
PGCOPYDB_DROP_IF_EXISTS=on pgcopydb restore pre-data --no-owner --resume --not-consistent
pgcopydb copy table-data --resume --not-consistent
pgcopydb copy indexes --resume --not-consistent
```
the last command (indexes) fails on both this and main branch at firs run as follows:

```bash
09:12:04.390 50533 INFO   Running pgcopydb version 0.15.74.gc74047a from "/home/cem/scratch/pgcopydb/src/bin/pgcopydb/pgcopydb"
09:12:04.392 50533 INFO   [SOURCE] Copying database from "postgres://cem@127.0.0.1:5434/testdb1?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60"
09:12:04.392 50533 INFO   [TARGET] Copying database into "postgres://cem@127.0.0.1:5435/testdb2?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60"
09:12:04.422 50533 INFO   Using work dir "/tmp/pgcopydb"
09:12:04.424 50533 INFO   Re-using catalog caches
09:12:04.424 50533 INFO   Creating 1 indexes in the target database using 4 processes
09:12:04.436 50542 INFO   STEP 6: starting 4 CREATE INDEX processes
09:12:04.437 50542 INFO   STEP 7: constraints are built by the CREATE INDEX processes
09:12:04.478 50543 INFO   Found 1 indexes on target database for table public.table1
09:12:04.489 50543 ERROR  Failed to send a message to (null) queue (0) with type 1: Invalid argument
09:12:04.490 50543 ERROR  Failed to queue VACUUM ANALYZE public.table1 [16386]
09:12:04.490 50543 ERROR  Failed to create index with oid 16390, see above for details
09:12:04.493 50543 ERROR  CREATE INDEX worker 50543 encountered 1 errors, see above for details
09:12:04.538 50542 ERROR  Sub-process 50543 exited with code 12
09:12:04.539 50542 ERROR  Some INDEX worker process(es) have exited with error, see above for details
09:12:04.539 50542 ERROR  Failed to create indexes, see above for details
09:12:04.625 50533 ERROR  Sub-process 50542 exited with code 12
09:12:04.626 50533 ERROR  Some sub-processes have exited with error status, see above for details
```
but the second run succeeds as follows:
```bash
09:36:55.946 51658 INFO   Running pgcopydb version 0.15.74.gc74047a from "/home/cem/scratch/pgcopydb/src/bin/pgcopydb/pgcopydb"
09:36:55.951 51658 INFO   [SOURCE] Copying database from "postgres://cem@127.0.0.1:5434/testdb1?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60"
09:36:55.951 51658 INFO   [TARGET] Copying database into "postgres://cem@127.0.0.1:5435/testdb2?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60"
09:36:56.026 51658 INFO   Using work dir "/tmp/pgcopydb"
09:36:56.031 51658 INFO   Re-using catalog caches
09:36:56.031 51658 INFO   Creating 1 indexes in the target database using 4 processes
09:36:56.044 51667 INFO   STEP 6: starting 4 CREATE INDEX processes
09:36:56.045 51667 INFO   STEP 7: constraints are built by the CREATE INDEX processes
```
What is your take on it? Is it expected? Are the commands valid or invalid?

Reply:
Commands look valid to me. So that's another bug to report and fix.

Reply:
Opened a new issue #813